### DISCUSSION TITLE
I have 2 questions GGUF and Preview lora

### DISCUSSION THREAD

**Discussion by SunGreen777** [Comfy-Org/ComfyUI#5804] [General]
I used A1111 for a long time, then Forge. In Forge I used GGUF models, comfy desktop does not see them, is it supposed to be like this?
I saw images from Preview lora on comfy desktop presentation videos, how do I enable it?

**Comment by ltdrdata** [comment]
You need to install custom nodes.
https://github.com/city96/ComfyUI-GGUF
https://github.com/Kinglord/ComfyUI_LoRA_Sidebar

**Reply by SunGreen777** [reply]
> You need to install custom nodes. https://github.com/city96/ComfyUI-GGUF https://github.com/Kinglord/ComfyUI_LoRA_Sidebar
Thank you, I probably overestimated the Desktop version, I expected it to automatically load the missing nodes.
Don’t you know about prewiev?

**Reply by ltdrdata** [reply]
You can see missing nodes from here.
![image](https://github.com/user-attachments/assets/e35152cd-3bc9-4844-9b7a-e84a616608ae)
![image](https://github.com/user-attachments/assets/5a90a177-42cd-4906-bde6-ea2c3de8d0c6)

**Reply by ltdrdata** [reply]
It is possible to suggest missing nodes, but they should not be installed automatically. This could lead to security vulnerabilities.
